test/types/index.tst.ts

```diff
@@ -1,87 +1,121 @@
-import { expect, test } from 'tstyche'
+import { describe, expect, test } from 'tstyche'
 
-import { InternalTypedDocument, Orama, PartialSchemaDeep, Results, Schema, SearchParams, TypedDocument } from '@orama/orama'
+import { type InternalTypedDocument, type Orama, type PartialSchemaDeep, type Results, type Schema, type SearchParams, type TypedDocument } from '@orama/orama'
 import Fastify from 'fastify'
 import fp from 'fastify-plugin'
 
-import { PersistenceInFile, PersistenceInMemory, fastifyOrama } from '../../index.js'
-
-const app = Fastify()
+import { OramaApi, PersistenceInFile, PersistenceInMemory, fastifyOrama } from '../../index.js'
 
 const mySchema = {
   quote: 'string',
   author: 'string'
 } as const
+
 type MySchema = Schema<typeof mySchema>
 
+const baseSetup = () => {
```

Comment:
(nit+optional): function notations are prefered 

```suggestion
function baseSetup() {
```